Accessibility active learning updates (#40327)
mdn/content: files/en-us/learn_web_development/core/accessibility/test_your_skills/html/index.md

> In this task we will test your understanding of text semantics, and why they are good for accessibility. The given text is an information panel with action buttons, but the HTML is really bad.

Reviewer: ```suggestion
In this task we will test your understanding of semantic HTML, and why it is good for accessibility. The given text is an information panel with action buttons, but the HTML is really bad.
```

  <a href="/education">click here</a>), sponsorship pack (
  <a href="/resources/sponsorship.pdf">click here</a>), and assessment sheets (
  <a href="/resources/assessments.docx">click here</a>).
</p>
```

{{ EmbedLiveSample("html-ally-3", "100%", 140) }}

<details>
<summary>Click here to show the solution</summary>

Your finished HTML should look something like this:

```html
<p>
  For more information about our activities, check out our
  <a href="/fundraising">fundraising page</a>,
  <a href="/education">education page</a>,
  <a href="/resources/sponsorship.pdf">sponsorship pack (PDF, 8MB)</a>, and
  <a href="/resources/assessments.docx">assessment sheets (Word document)</a>.
</p>
```

</details>

## HTML accessibility 4

In our final HTML accessibility task, you are given an image gallery, which has some accessibility problems. Can you fix them?

- The header image has an accessibility issue, and so do the gallery images.
- You could take the header image further and implement it using CSS for arguably better accessibility. How would you create such a solution?

To complete the task:

1. Click **"Play"** in the code block below to edit the example in the MDN Playground.

> 2. Update the code to fix the problems described earlier.

Reviewer: Here and elsewhere. It makes no sense to refer to earlier when the comment is immediately above in the same section. 

```suggestion
2. Update the code to fix the problems described above.
```

You could leave it because it aligns with the spirit of the guideline for code examples discussed here: https://github.com/mdn/content/pull/38903/files#r2027933628. Though as you can see from the discussion I very strongly disagree with sacrificing clarity in order to signal we care about inclusivity (unless it actually does contribute). Meh.

Author: I agree with you, and I'm happy to change this. Done.
